packages/vscode-extension/src/debugging/DebugAdapter.ts

```diff
@@ -143,7 +149,64 @@ export class DebugAdapter extends DebugSession {
     }
   };
 
+  async formatDefaultString (
```

Comment:
Based `formatDefaultString` and `formatComplexStringOutput` based on https://github.com/microsoft/vscode-js-debug/blob/9a436b1a3060958110959c590eac9d1a8d548922/src/adapter/console/textualMessage.ts#L83